backend/app/models/experiment_run.py

```diff
@@ -107,8 +107,8 @@ def is_editable_by_user(self, user: dict | None) -> bool:
     def retry_failed_run(self):
         return ExperimentRun(
             created_by=self.created_by,
```

Comment:
This is incorrect. We are not creating a DTO, so the attribute name remains the same (`is_public=self.is_public`)